spec/controllers/controller_patches_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "caching blog content" do
 
     before do
-      allow(controller).to receive(:blog).and_return("")
+      allow(controller).to receive(:get_blog_content)
```

Comment:
ref: [get_blog_content](https://github.com/mysociety/alaveteli/blob/develop/app/controllers/general_controller.rb) in the current Alaveteli repo